corecomponent/androidcomponent/src/main/res/values/colors.xml

```diff
@@ -1,17 +1,58 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <!-- Theme -->
```

Comment:
I arranged colors.xml so that we can support dark mode by just adding `values-night/colors.xml`.